README.md

```diff
@@ -1,2 +1,12 @@
 # HtmlCssLinter
 Microverse HTML and CSS project with Linter.
+
+🛠 Built With
+
+Tech Stack
+    HTML
+    CSS
+
+🚀 Live Demo
+
+    Want to see the page:) => https://ab-noori.github.io/HtmlCssLinter
```

Comment:
- [ ] **As suggested by the previous reviewer,** Kindly remember that the README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo. 😎😎😎